var/spack/repos/builtin/packages/openfast/package.py

```diff
@@ -17,29 +17,29 @@ class Openfast(CMakePackage):
 
     version("develop", branch="dev")
     version("master", branch="main")
-    version("4.0.2", tag="v4.0.2")
-    version("3.5.5", tag="v3.5.5")
-    version("3.5.4", tag="v3.5.4")
-    version("3.5.3", tag="v3.5.3")
-    version("3.4.1", tag="v3.4.1")
-    version("3.4.0", tag="v3.4.0")
-    version("3.3.0", tag="v3.3.0")
-    version("3.2.1", tag="v3.2.1")
-    version("3.2.0", tag="v3.2.0")
-    version("3.1.0", tag="v3.1.0")
-    version("3.0.0", tag="v3.0.0")
-    version("2.6.0", tag="v2.6.0")
-    version("2.5.0", tag="v2.5.0")
-    version("2.4.0", tag="v2.4.0")
-    version("2.3.0", tag="v2.3.0")
-    version("2.2.0", tag="v2.2.0")
-    version("2.1.0", tag="v2.1.0")
-    version("2.0.0", tag="v2.0.0")
-    version("1.0.0", tag="v1.0.0")
+    version("4.0.2", tag="v4.0.2", commit="fc1110183bcc87b16d93129edabdce6d30e3a497")
+    version("3.5.5", tag="v3.5.5", commit="b48e032303c12df6949c67ebffb7050b888db000")
+    version("3.5.4", tag="v3.5.4", commit="6a63db76978b0f703165391291963beb517d5a9e")
+    version("3.5.3", tag="v3.5.3", commit="6a7a543790f3cad4a65b87242a619ac5b34b4c0f")
+    version("3.4.1", tag="v3.4.1", commit="18704086dad861ab13daf804825da7c4b8d59428")
+    version("3.4.0", tag="v3.4.0", commit="e8ec53f9c7f9d3f6a13bfb61dba12a0ca04d8a2f")
+    version("3.3.0", tag="v3.3.0", commit="5f3fb6ef74f48e75ca94000090737a41866fb264")
+    version("3.2.1", tag="v3.2.1", commit="08fffef240461a8334596179f1de462be43ad3e9")
+    version("3.2.0", tag="v3.2.0", commit="90a1ffb626baf398d89681b9422bdbfef11cd3ad")
+    version("3.1.0", tag="v3.1.0", commit="3456a645581456883e44d441eb285ed688e98797")
+    version("3.0.0", tag="v3.0.0", commit="42a5a8196529ae0349eda6d797a79461c2c03ff0")
+    version("2.6.0", tag="v2.6.0", commit="bbbb1ca7b28a4ba411613b5c85f5de02f8316754")
+    version("2.5.0", tag="v2.5.0", commit="718d46f707d78e85edf1b49d3b1a63e8e23e1aae")
+    version("2.4.0", tag="v2.4.0", commit="ff33ca1cf65f2e13c1de0ab78cc2396ec4a47ce0")
+    version("2.3.0", tag="v2.3.0", commit="f2419c5d1c23caad9146b95a103d89e9dcaefe30")
+    version("2.2.0", tag="v2.2.0", commit="e4faf27b774982df274b87c0570e4b58c4a13fe3")
+    version("2.1.0", tag="v2.1.0", commit="f147b80521eff90c19f065eabeceac13de39ac59")
+    version("2.0.0", tag="v2.0.0", commit="0769598a17e19b3ccd00a85cde389995f55024a8")
+    version("1.0.0", tag="v1.0.0", commit="e788b9b18bd5ed96ea59d4bc0812d461bc430cfe")
```

Comment:
Same question as the amr-wind one.

Reply:
I would also deprecate anything below 3.0.